R/checks.R

```diff
@@ -90,18 +112,21 @@ check_host = function(auto_connect=FALSE,verbose=FALSE){
   return(TRUE)
 }
 
-check_times = function(relative_paths,archive_mode=FALSE,force_backup=FALSE){
+
+check_times = function(relative_paths,
+                       archive_mode = FALSE,
+                       force_backup = FALSE){
   
-  local_base = base_path=config::get("project_base")
-  remote_base = base_path=config::get("project_base",config="default")
+  local_base = base_path = check_config_value(config::get("project_base"))
```

Comment:
For this, I just added the newly added `check_config_value` to any **existing** line that gets paths from the config. So, if this indeed is something funny going on here, it would be outside the scope of what's introduced in this PR. But should be addressed nonetheless.

Reply:
Wow. What does this line actually accomplish? I'm surprised R doesn't complain about this syntax.

Reply:
This line creates two variables (local_base and base_path) pointing to the same path ("/projects/nhl_meta_analysis_scratch/gambl/results_local/"). 

Yeah, it's kinda strange that R accepts this. 

From the looks of it, this double assignment was added 7 months ago by Ryan. 

I will drop the `base_path` part since it, from the looks of it, doesn't do anything in the function. 

Sounds good?

Reply:
Yes it must be a pasting error. If you see something like this with weird syntax again please consider it a likely bug and fix it if you can.